modules/breakpad/SCsub

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_breakpad = env_modules.Clone()
+
+# Thirdparty source files
+
+thirdparty_obj = []
+
+thirdparty_dir = "#thirdparty/breakpad/"
+
+# TODO: find out when these are needed (if at all)
+dwarf_module = False
+stabs_module = False
```

Comment:
Did anyone find that these were needed in the end? If not, I'd remove the relevant code (and perhaps keep a second branch on your fork with this code present).

Reply:
If I remember right, no one has tried this out on mac yet. So there is a chance that these would be required on mac.

If it is determined necessary to get this PR merged, I now have a mac I can use for development so I can check things there to make sure the mac version compiles and can create crash dumps.